diff --git a/common/cmake/package.cmake b/common/cmake/package.cmake
index daa4f49fa..4e9abd61b 100644
--- a/common/cmake/package.cmake
+++ b/common/cmake/package.cmake
@@ -118,7 +118,7 @@ ENDIF()
 # Install MSVC runtime
 ##############################################################
 
-IF (WIN32)
+IF (0)
 
   SET(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP TRUE)
   SET(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_NO_WARNINGS TRUE)
@@ -348,4 +348,4 @@ add_custom_target(
   COMMAND cd ${PROJECT_SOURCE_DIR}/tests/integration/test_embree_release && ${CMAKE_COMMAND} -B build --preset ${EMBREE_TESTING_INTEGRATION_PRESET} -Dembree_DIR="${CMAKE_CURRENT_BINARY_DIR}/embree_install/lib/cmake/embree-${EMBREE_VERSION}"
   COMMAND cd ${PROJECT_SOURCE_DIR}/tests/integration/test_embree_release && ${CMAKE_COMMAND} --build build --config Release
   COMMAND cd ${PROJECT_SOURCE_DIR}/tests/integration/test_embree_release && ${INTEGRATE_BINARY}
-)
\ No newline at end of file
+)
